src/main/java/org/prebid/server/proto/openrtb/ext/request/liftoff/ExtImpLiftoff.java

```diff
@@ -0,0 +1,19 @@
+package org.prebid.server.proto.openrtb.ext.request.liftoff;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder(toBuilder = true)
+@Getter
```

Comment:
@Value

There is only three params, can be done with `of` static method

Reply:
In our code we should update `ExtImpLiftoff` obj field.